### ISSUE TITLE
Make it possible to easily view debug logs within the app

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-ios#776]
### Expected behavior

With the recent launch of the [Pocket Casts Forum](https://forums.pocketcasts.com/), we want our users to be able to view their logs easily without having to make a support ticket or go through that flow. It would be great if we could do something similar to the WordPress apps and add an option under Settings to view/export them

**Comment by rviljoen** [comment]
I agree that this can be improved, but there is a somewhat convoluted way to view the logs without logging a ticket:

Profile->Settings->Help&Feedback->(Scroll to bottom)->Get in touch->Get Support
On this screen, you can click on _"Attached Logs"_ and scroll to _"Debug Logs"_ to see if you can spot anything weird happening at that time.

Making an easily accessible view/export button will be a great addition.

**Comment by joashrajin** [comment]
Yep, I’m aware of the current process but we need an easier way to view it going forward :)

On Android it’s even trickier, so if we could add options in the app to view it easily, it should help a lot

**Comment by mylesbarros** [comment]
@joashrajin Right now we show a web view with a FAQ with the contact options at the bottom.

Would the concept be to mimic Wordpress where tapping *Help & Feedback* takes us to a Help screen with a link to the community forums and the logs and an additional button to bring up the existing FAQ web view?
